plugin/mutate/source/dextool/plugin/mutate/backend/report/html/tmpl.d

```diff
@@ -285,6 +285,11 @@ struct TimeScalePointGraph {
             return d;
         }();
 
+        if (name == "ScoreByCodeChange") {
```

Comment:
reconsider this implementation. `TimeScalePointGraph` is generic and shouldn't change its behavior based on a magic `name` parameter.